diagnostic.js

```diff
@@ -30,25 +30,68 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+'use strict';
+
+const models = require('app/models');
+const Movement = models.movement;
```

Comment:
If this were inside Antony's template, this code would work, but in a bare app like this it unfortunately will not - you'll need to explicitly require the file (with path) where your model lives.